samples/cpp/classification_sample_async/main.cpp

```diff
@@ -81,22 +82,10 @@ int main(int argc, char* argv[]) {
         // -------- Step 1. Initialize OpenVINO Runtime Core --------
         ov::runtime::Core core;
 
-        if (!FLAGS_l.empty()) {
-            auto extension_ptr = std::make_shared<InferenceEngine::Extension>(FLAGS_l);
-            core.add_extension(extension_ptr);
```

Comment:
User can add new extensions if model contains custom operations

Reply:
sample is not about that, is not it?